xml/templates/message_templates/case_activity_text.tpl

```diff
@@ -2,16 +2,16 @@
 {ts}Activity Summary{/ts} - {$activityTypeName}
 ===========================================================
 {if !empty($isCaseActivity)}
-{ts}Your Case Role(s){/ts} : {$contact.role}
-{if $manageCaseURL}
+{ts}Your Case Role(s){/ts} : {$contact.role|default:''}
```

Comment:
I hadn't realised this was an option

Reply:
yeh @totten raised it here https://chat.civicrm.org/civicrm/pl/jbbd4efzy3yp7ed9y1bjnphnuy

Reply:
Given this test specifically looks for Your Case Role in the output figured the default was the better way to go